.husky/prepare-commit-msg

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+exec < /dev/tty && node_modules/.bin/cz --hook || true
```

Comment:
I don't understand what this does 😬 

I get what `cz` is but what does this whole command do in unison?

Reply:
Doesn't commitizen already prevent commits from straying from conventions?

Reply:
`exec < /dev/tty` opens a filesystem object representing the terminal process. If that succeeds then, ` node_modules/.bin/cz --hook` runs the commitizen. `|| true` checks if the entire `exec < /dev/tty && node_modules/.bin/cz --hook` execution succeed, if not it'll output `true`.

Commitizen does prevent commits from straying from convention after the commit message has been submitted such as `git commit -m "commitizen sucks"`. However, the husky/git hook that I added will force the user whenever they run `git commit` or `git commit -m "some messager"` to fill out a commitizen template in the terminal.

The `prepare-commit-msg` hook is for enforcing commitizen and the `pre-commit` hook is for checking the commit meets commitizen/commit conventions